src/Sylius/Bundle/ApiBundle/Command/ContactAwareInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+interface ContactAwareInterface extends CommandAwareDataTransformerInterface
```

Comment:
you need this interface for data transformer, but there you are using only `email` so firstly IMO `getMessage` shouldn't be in this interface secondly I have a problem with this name, but also I don't have any idea for now how to improve it. maybe @GSadee or @lchrusciel can help :D